Utils/gen-unicode-data/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
+DerivedData/
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
+Package.resolved
```

Comment:
What's up with this? Why doesn't it just inherit from the package parent folder?